### PR TITLE
Support ufuncs in combination with PyPy.

### PR DESCRIPTION
**Pull request by ghuls**

### CODE REVIEW

File: py-polars/src/series.rs [hunk 1]
```diff
@@ -1379,17 +1379,50 @@ macro_rules! impl_ufuncs {
                 let (out_array, av) =
                     unsafe { aligned_array::<<$type as PolarsNumericType>::Native>(py, size) };
 
-                debug_assert_eq!(out_array.get_refcnt(), 1);
+                debug_assert_eq!(
+                    {
+                        let refcnt = out_array.get_refcnt();
```
Comment (ritchie46) [inline py-polars/src/series.rs:1384]: Shall we make a `get_refcnt` function to have this logic in one place.

Comment (ghuls) [reply py-polars/src/series.rs:1384]: Probably a better idea, yeah

Comment (ghuls) [reply py-polars/src/series.rs:1384]: where should this function be put?

Comment (ritchie46) [reply py-polars/src/series.rs:1384]: Let's go for `utils.rs` for now. Later we can create a python module where we keep this kind of stuff.

Comment (ghuls) [reply py-polars/src/series.rs]: I put it in `npy.rs` as for now get_refcnt only supports `PyArray1` types.

**Pull request merged**